docs/public-networks/how-to/use-configuration-file/profile.md

```diff
@@ -1,82 +1,41 @@
 ---
-sidebar_position: 1
+title: Use a profile 
+sidebar_position: 2
 ---
 
-# Use a profile
+# Use a profile 
 
-To help you get started quickly, Besu provides pre-configured profiles containing custom settings
-for some common use cases.
-You can load these profiles using the [`--profile`](../../reference/cli/options.md#profile) CLI option.
-For example:
+Load a profile using the [`--profile` CLI option](../../reference/cli/options.md#profile).
 
-```bash
-besu --config-file=config.toml --profile=staker
-```
-
-In this example, `config.toml` is the user-provided [configuration file](index.md), and `staker` is
-the pre-configured profile containing custom settings.
-
-Any configuration options explicitly set in the configuration file or command line will overwrite
-the same options set in the profile.
-See the [configuration order of precedence](index.md#configuration-order-of-precedence) for more information.
-
-You can use the following profiles:
+You can optionally use profiles to extend Besu's configuration. Combined with the [boilerplate configuration](defaults.md), profiles simplify the process of applying viable defaults. Besu's pre-configured profiles optimize for the following supported use cases:
 
 - [Minimalist staker profile](#minimalist-staker-profile)
 - [Staker profile](#staker-profile)
 - [Enterprise/Private profile](#enterpriseprivate-profile)
 
+
+:::note
+
+A configuration explicitly set in the configuration file or command line will 
+[override the same option/s](index.md#configuration-order-of-precedence) set in the profile.
+
+:::
+
 ## Minimalist staker profile
 
-For stakers who want to maximize their hardware value but don't want to serve full sets of data to
-their peers, Besu provides a minimalist staker profile.
-See the
+[`--profile=MINIMALIST_STAKER`](../../reference/cli/options.md#profile) is optimized for stakers who want to maximize their hardware value but don't want to serve full sets of data to their peers, See the
 [minimalist staker profile on GitHub](https://github.com/hyperledger/besu/blob/8b64023a121ea996ef60e4b7e2299c5807683f90/config/src/main/resources/profiles/minimalist-staker.toml)
 for the custom settings.
 
-To use the minimalist staker profile, run Besu with
-[`--profile`](../../reference/cli/options.md#profile) set to `minimalist_staker`:
-
-```bash
-besu --profile=minimalist_staker
-```
-
 ## Staker profile
 
-For stakers who want to maximize their hardware value and also want to serve full sets of data to
-their peers, Besu providers a staker profile.
-See the
+[`--profile=STAKER`](../../reference/cli/options.md#profile) is optimized for stakers who want to maximize their hardware value while also serving full sets of data to their peers. See the
 [staker profile on GitHub](https://github.com/hyperledger/besu/blob/8b64023a121ea996ef60e4b7e2299c5807683f90/config/src/main/resources/profiles/staker.toml)
```

Comment:
This permalink could also out of date, suggest using https://github.com/hyperledger/besu/blob/main/config/src/main/resources/profiles/staker.toml

Reply:
agree/applied